src/Row.ts

```diff
@@ -37,15 +37,44 @@ class Row extends RowBase<RowProperties> {
 				classes: this.classes(tableCss.table, css.rowTable)
 			}, [
 				v('tr', columns.map((column) => {
-					const { id, field } = column;
+					const { field, id } = column;
+
+					let value: any;
+					if (column.get) {
+						// Get the value from the column callback
+						value = column.get(item, column);
+					}
+					else if (!column.render) {
+						// Get the value using a property lookup on the item data
+						value = item.data[ field || id ];
+					}
+					else {
+						// This column has no value
+						value = '';
+					}
+
+					let content: DNode;
+					if (column.render) {
```

Comment:
With the `options` I'd just expect consumers to destructure what is needed at the top of their custom renderer function (which I think is ergonomic).

```ts
const { value } = options;
```

Both `get` and `render` return a `DNode` and are used to effectively populate `content` (i'm not sure why cell needs `value` now there is `content`) that is used by `Cell`? Just feels like having two functions that end up as the same thing - `content` is more confusing than flexible (would be for me), because I can return a string either based on item and column.

So just having a API for `customCellRenderer(item: any, column: Column): DNode {}`, should cover all the scenarios needed?

```ts
const { customCellRenderer, field = column.id } = column;
let content = item.data[ field ] ? String(item.data[ field ]) : '';
if (customCellRenderer) {
    content = customCellRenderer(item, column);
}
```

Means a consumer can return a 

1) Static string:

```ts
function customCellRenderer(item: any, column: Column): DNode {
     return 'static-string';
}
```

2) Dynamic string based on the item

```ts
function customCellRenderer(item: any, column: Column): DNode {
    const { field = column.id } = column;
    const value = item[field];
    return `Miss ${value}`;
}
```

3) A custom DNode

```ts
function customCellRenderer(item: any, column: Column): DNode {
    return v('button', { /* some properties */});
}
```